test/normalize.html

```diff
@@ -0,0 +1,493 @@
+<!DOCTYPE html>
```

Comment:
ah, if we already are checking the contents of `cases/normalize-reset.css` explicitly against a known correct value, then we shouldn't have to visually confirm that normalize is working. unless I'm misunderstanding this, then please correct me.